src/winetricks

```diff
@@ -5093,14 +5059,20 @@ winetricks_set_wineprefix()
             WINE64="$(dirname "${WINE}")/"
             [ "${WINE64}" = "./" ] && WINE64=""
             WINE64="${WINE64}$(basename "${WINE}" | sed 's/^wine/wine64/')"
+            test -x "${WINE64}" || WINE64="${WINE}"
+        fi
+
+        # if we can't detect wine64 command, it is the new wow64 mode
```

Comment:
The `wine-wow64` package on Arch AUR creates a `wine64` -> `wine` symlink, so this doesn't work in that situation:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wine-wow64#n100

No idea if other platforms do this, though. I imagine they might do it for backwards compatibility.

Maybe we could check if `WINE64` is a symlink to `WINE`?

Reply:
> The `wine-wow64` package on Arch AUR creates a `wine64` -> `wine` symlink, so this doesn't work in that situation: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wine-wow64#n100
> 
> No idea if other platforms do this, though. I imagine they might do it for backwards compatibility.
> 
> Maybe we could check if `WINE64` is a symlink to `WINE`?

I would prefer they remove wine64 link, but we can add workaround too (check if WINE64 is a symlink to WINE).

Reply:
I'm not sure how common this scenario is. Most distros don't seem to ship the new WoW64 yet (as in, the `--enable-archs=x86_64,i386` is set in the build script), so I have no idea how common this symlink is. Fedora, Ubuntu and Arch don't seem to ship the new WoW 64 build.

Reply:
On Gentoo Linux, we allow for multiple Wine versions to be installed at the same time, including different flavours (Vanilla, Staging, and Proton for example), so in general 'wine' will be a symlink to '/etc/eselect/wine/bin/wine' which will be a link to the currently selected 'main' provider.

We also have a USE-flag to toggle the new style 'wow64' build, which is not handled differently with regards to the links, and as such, this change will not help Winetricks to detect either.

May be tricksy to catch them all.  : ]

Reply:
This check needed only for text message. I can just remove it. Our near future is new WoW64 in any case.

Reply:
Well, I have added a patch to remove wine64 checking and a bit obsoleted warning message too.